modules/testing/src/views/full/index.tsx

```diff
@@ -1,16 +1,19 @@
 import { Icon, Intent } from '@blueprintjs/core'
 import { IconNames } from '@blueprintjs/icons'
 import { AxiosStatic } from 'axios'
-import { confirmDialog } from 'botpress/shared'
+import { confirmDialog, toast } from 'botpress/shared'
 import _ from 'lodash'
 import React from 'react'
 import { Grid, Row, Col, Button } from 'react-bootstrap'
+
 import { Preview, Scenario, Status } from '../../backend/typings'
 import NoScenarios from './NoScenarios'
 import ScenarioComponent from './Scenario'
 import ScenarioRecorder from './ScenarioRecorder'
 import style from './style.scss'
 
+const WEBCHAT_READY_TIMEOUT = 10000
```

Comment:
I opted for a 10sec timeout since I simply don't want to be waiting indefinitely if something goes wrong but at the same time I want to give the webchat all the time it need to create the new session.